Cargo.toml

```diff
@@ -9,6 +9,7 @@ exclude = ["/dlib-test", "README.tpl"]
 readme = "README.md"
 keywords = ["dylib", "dlopen"]
 categories = ["api-bindings"]
+edition = "2021"
```

Comment:
Is this specifically needed for this change?

Reply:
It shouldn't be necessary, no. It just seems a bit unusual at this point for a crate to be using the default edition.

I don't think there's any need to support Rust versions too old for the 2021 edition at this point.

Reply:
In that case, could you also update the CI accordingly? It's currently failing because of that change.

Reply:
I've updated CI to the first version of Rust that supports the 2021 edition.